lib/TopSitesFeed.jsm

```diff
@@ -35,8 +35,8 @@ const ROWS_PREF = "topSitesRows";
 // Search experiment stuff
 const NO_DEFAULT_SEARCH_TILE_EXP_PREF = "improvesearch.noDefaultSearchTile";
 const SEARCH_HOST_FILTERS = [
-  {hostname: "google", identifierPattern: /^google/},
-  {hostname: "amazon", identifierPattern: /^amazon/}
+  {hostname: "google", identifierPattern: /^google|^www.google/},
```

Comment:
The identifierPattern was originally to match the identity of the search engine in Firefox, which is a string like 'google' or 'google-2018' or 'amazon-en-GB'. You can see it used that way on line 52. It's bit confusing to also use it to match a hostname on line 61, maybe make the hostname a regex with `/^google|^www.google/` instead?

Reply:
Yeah I would keep this list separate so as not to affect the other experiment anyway